controller/task_manager.py

```diff
@@ -87,7 +87,7 @@ def start_rats_task(
     only_uploaded_attributes: bool = False,
     attribute_id: Optional[str] = None,
 ) -> int:
-    if tokenization.is_doc_bin_creation_running(project_id):
+    if tokenization.is_doc_bin_creation_running_or_queued(project_id, only_running=True):
```

Comment:
i think the only_running should be False as in the previous version or is there some specific reason that was changed to true?

Reply:
I have set `only_running` to True because of [function signature changes](https://github.com/code-kern-ai/refinery-submodule-model/commit/f1609aafa2a98cb3463fa828459f1057f2b9937c#diff-441c827430db4d2e9a0fc5190108c0d966845e66706513d0e2d1aee3e9f09970L222)